SimG4Core/Application/python/g4SimHits_cfi.py

```diff
@@ -37,12 +37,13 @@
 )
 
 common_UseLuminosity = cms.PSet(
-    InstLuminosity  = cms.double(0.),   
+    InstLuminosity  = cms.double(0.),
     DelivLuminosity = cms.double(5000.)
 )
 
 common_MCtruth = cms.PSet(
     DoFineCalo = cms.bool(False),
+    SaveCaloBoundaryInformation = cms.bool(False),
```

Comment:
@rovere , I am sorry, can you imagine that somebody want DoFineCalo=True and SaveCaloBoundaryInformation=False

What is the use case?

Reply:
Ciao @civanch 
thanks for looking into that!
The use case here is that we would like to have the information at the caloBoundary without requesting the full complexity of tracing everything within the fine-calo volume.
The two flags, in this sense, are complementary.

Reply:
OK

Reply:
> @rovere , why we see differences in Phase-2 WFs? I would expect regression for this PR. DD4Hep WF may be fine so far.

ciao @civanch 
the flag is set to `false` for all workflows but for `Phase2` ones.
The changes are mostly coming from SimTracks that are now saved if they have the crossedBoundary flag set. I found this useful to correctly assign hits to SimTrack and SimClusters.
The extent of the changes is rather limited and of limited impact on the high level Validation.

The DD4HEP workflows shows many differences, but I'm not sure where they are coming from, but you seem to suggest that is fine, right?